it/auth/IndividualsStateBenefitsAuthSupportingAgentsAllowedISpec.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import api.models.domain.TaxYear
+import api.services.DownstreamStub
+import play.api.http.Status.OK
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+
+class IndividualsStateBenefitsAuthSupportingAgentsAllowedISpec extends AuthSupportingAgentsAllowedISpec {
+
+  val callingApiVersion = "1.0"
+
+  val supportingAgentsAllowedEndpoint = "create-benefit"
+
+  private val taxYear = TaxYear.fromMtd("2019-20")
+
+  val mtdUrl = s"/$nino/${taxYear.asMtd}"
+
+  val requestBodyJson: JsValue = Json.parse(
```

Comment:
This is only used in 1 place, so could just be moved so it's inside the "def sendMtdRequest()" method (just make the method multi-line)